Board.js

```diff
@@ -0,0 +1,27 @@
+class Board {
+  constructor(length) {
+    this.board = new Array(length).fill("_");
+  }
+  length() { 
+    return this.board.length;
+  }
+
+  get(idx) {
+  return this.board[idx];
+  }
+
+  isComplete() {
+    return this.board.every(element => element !== "_");
+     
+  
+  addLetter(word, letter) {
```

Comment:
This method is not being used anywhere, i was getting an error until i commented this out